fs/stat.c

```diff
@@ -49,17 +69,18 @@ static struct fd *at_fd(fd_t f) {
     return f_get(f);
 }
 
-static dword_t sys_stat_path(fd_t at_f, addr_t path_addr, addr_t statbuf_addr, bool follow_links) {
+// The `flags` parameter accepts AT_ flags
+static dword_t sys_stat_path(fd_t at_f, addr_t path_addr, addr_t statbuf_addr, int flags) {
```

Comment:
This function now has the exact signature of sys_fstatat64 so maybe rename to sys_fstatat64